### DISCUSSION TITLE
Upgrade to .NET 8?

### DISCUSSION THREAD

**Discussion by plettb** [Azure/azure-mobile-apps#931] [Q&A]
I'm trying to get authentication working in Maui.  The sample provided is still in .NET 6, which produces `Error (active)	NETSDK1202	The workload 'net6.0-android' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/maui-support-policy for more information about the support policy.	TodoApp.MAUI (net6.0-android)`, and, obviously, won't build nor run.

I'd really appreciate someone taking the time to upgrade it to .NET 8.

**Comment by adrianhall** [comment]
The “azure-mobile-apps” version will not get upgraded to .NET 8.

I have moved all development over to CommunityToolkit/Datasync - where I’m currently implementing a .NET 8 client.  However, it’s not complete yet.
